packages/world/src/TreeLib.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+/* Auto‑generated. DO NOT EDIT. */
+pragma solidity >=0.8.24;
+
+import { ObjectTypeId } from "./ObjectTypeId.sol";
+import { ObjectTypes } from "./ObjectTypes.sol";
+import { Vec3 } from "./Vec3.sol";
+
+library TreeBlobs {
+  /* Oak – 50 fixed, 8 random */
```

Comment:
fixed == all oak trees will have 50 leaves
random = an oak tree can have 0-8 additional leaves

oak tree will have 50-58 leaves

is that correct?

Reply:
yup